cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,85 @@
-/// <reference types="cypress" />
-/// <reference types="../support" />
+import homePageObject from '../support/pages/home.pageObject';
```

Comment:
If you can't fix error fix test please ask a question about it in Slack or join QnA session